### DISCUSSION TITLE
How to populate a new fidb file?

### DISCUSSION THREAD

**Discussion by TimRegeant** [NationalSecurityAgency/ghidra#6876] [General]
I have an existing project and want to create FunctionIDs for some of the static library calls in it but I don't have an existing FIDB file.

So I created a new file with Tools->Function ID-> Create New Empty FidDb...

I have a LIB directory with numerous .A files.  How do I populate these into the fidb I created?

When I choose  Tools->Function ID->Populate fidDb from programs...  It opens a dialog and wants me to select some file from the root project folder which is not where the library files are.  Do I need to use headless mode somehow?  

Any help is appreciated.

**Comment by TimRegeant** [comment]
question irrelevant now.

**Reply by dlenski** [reply]
What changed? I am trying to do the same thing and encountering similar problems.
